sources/Renderer/Direct3D11/Shader/Builtin/Compile.bat

```diff
@@ -1,10 +1,10 @@
 REM Compiles all shaders with fxc
 REM Use VisualStudio command line tools to run this batch script
 
-fxc /D TEXTURE_DIM=1 /T cs_5_0 /E CopyTextureFromBuffer /Fo CopyTextureFromBufferCS.Dim1D.cso CopyTextureFromBuffer.hlsl
-fxc /D TEXTURE_DIM=2 /T cs_5_0 /E CopyTextureFromBuffer /Fo CopyTextureFromBufferCS.Dim2D.cso CopyTextureFromBuffer.hlsl
-fxc /D TEXTURE_DIM=3 /T cs_5_0 /E CopyTextureFromBuffer /Fo CopyTextureFromBufferCS.Dim3D.cso CopyTextureFromBuffer.hlsl
+fxc /D TEXTURE_DIM=1 /T cs_5_0 /E CopyTextureFromBuffer /Fh CopyTextureFromBufferCS.Dim1D.inc /Vn CopyTextureFromBufferCS_Dim1D CopyTextureFromBuffer.hlsl
```

Comment:
At first I thought we could use LLGL's `ReadFileAsHexStringDecl` script to generate the same format for the header files as for the Metal backend, but I guess using FXC's only capability to generate a C header is just fine and convenient.

I would like to have the same naming convention here for the extension, though. So changing this from `.inc` to `.cso.inl` would make it clear it's an inline-only header (with no header guard) for a CSO (Compute Shader Object).

Speaking of naming convention: FXC actually generates the default name in the same fashion as I have my global variables/constants with the `g_` prefix, but since FXC only uses the entry point name, we do in fact have to override the output name. Just adding a `g_` prefix to the output name (for `/Vn`) would follow the same pattern.

Could you please also add `/Qstrip_reflect` to the FXC command? I didn't realize how much bigger those resource files become but the text format made it more obvious.

Reply:
Whammy! Got those suggestions implemented real fast.